libraries/tlv-account-resolution/src/account.rs

```diff
@@ -15,12 +15,17 @@ use {
 
 /// Resolve a program-derived address (PDA) from the instruction data
 /// and the accounts that have already been resolved
-fn resolve_pda(
+fn resolve_pda<'a, A, G>(
```

Comment:
I chose to make it `G` to be distinct about the two closure types: `F`: _fetch_ account data (async), `G`: _get_ account data.

But if you think it should still be `F` that's totally fine!

Reply:
It can go either way I think, typically in math papers, `f` is the first letter used, and then `g`, so you can have `f(g(x))` or whatever. I have a slight preference for `F` here, but it's not a big deal